setup.cfg

```diff
@@ -24,6 +24,9 @@ branch = True
 source = howfairis
 
 [tool:pytest]
+python_files = test_*.py
+python_classes = Test*
+python_functions = test_*
```

Comment:
Is this stilll needed, partly fixed in another PR (#147 ) with lower undersoring of modules

Reply:
removed